marklogic-data-hub-central/ui/src/components/sidebar/sidebar.test.tsx

```diff
@@ -4,47 +4,70 @@ import Sidebar from './sidebar';
 import { entityFromJSON, entityParser } from '../../util/data-conversion';
 import modelResponse from '../../assets/mock-data/model-response';
 import searchPayloadFacets from '../../assets/mock-data/search-payload-facets';
+import { render, fireEvent } from '@testing-library/react';
+
 
 describe("Sidebar component", () => {
-    let wrapper;
-    const parsedModelData = entityFromJSON(modelResponse);
-    const entityDefArray = entityParser(parsedModelData);
-
-    describe('All Entities Dropdown Option', () => {
-      beforeEach(() => {
-        wrapper = shallow(
-          <Sidebar
-            entityDefArray={entityDefArray}
-            facets={searchPayloadFacets}
-            selectedEntities={[]}
-            facetRender = {jest.fn()}
-            checkFacetRender = {jest.fn()}
-          />
-        )
-      });
+  let wrapper;
+  const parsedModelData = entityFromJSON(modelResponse);
+  const entityDefArray = entityParser(parsedModelData);
 
-      it('should render only Hub Properties Panel', () => {
-        expect(wrapper.exists('#hub-properties')).toBe(true);
-        expect(wrapper.exists('#entity-properties')).toBe(false);
-      });
+  describe('All Entities Dropdown Option', () => {
+    beforeEach(() => {
+      wrapper = shallow(
+        <Sidebar
+          entityDefArray={entityDefArray}
+          facets={searchPayloadFacets}
+          selectedEntities={[]}
+          facetRender={jest.fn()}
+          checkFacetRender={jest.fn()}
+        />
+      )
     });
 
-    describe('An Entity is selected', () => {
-      beforeEach(() => {
-        wrapper = shallow(
-          <Sidebar
-            entityDefArray={entityDefArray}
-            facets={searchPayloadFacets}
-            selectedEntities={['Customer']}
-            facetRender = {jest.fn()}
-            checkFacetRender = {jest.fn()}
-          />
-        )
-      });
+    it('should render only Hub Properties Panel', () => {
+      expect(wrapper.exists('#hub-properties')).toBe(true);
+      expect(wrapper.exists('#entity-properties')).toBe(false);
+    });
+  });
 
-      it('should render both Entity and Hub Properties panel', () => {
-        expect(wrapper.exists('#hub-properties')).toBe(true);
-        expect(wrapper.exists('#entity-properties')).toBe(true);
-      });
+  describe('An Entity is selected', () => {
+    beforeEach(() => {
+      wrapper = shallow(
+        <Sidebar
+          entityDefArray={entityDefArray}
+          facets={searchPayloadFacets}
+          selectedEntities={['Customer']}
+          facetRender={jest.fn()}
+          checkFacetRender={jest.fn()}
+        />
+      )
     });
+
+    it('should render both Entity and Hub Properties panel', () => {
+      expect(wrapper.exists('#hub-properties')).toBe(true);
+      expect(wrapper.exists('#entity-properties')).toBe(true);
+    });
+  });
+
```

Comment:
I think we can remove beforeEach and the enzyme test lines 35-51, since the RTL test below verifies the same test better.

Reply:
Done. Thanks.